.repo-metadata.json

```diff
@@ -2,14 +2,15 @@
     "name": "pandas-gbq",
     "name_pretty": "Google BigQuery connector for pandas",
     "product_documentation": "https://cloud.google.com/bigquery",
-    "client_documentation": "https://pandas-gbq.readthedocs.io/en/latest/",
+    "client_documentation": "https://googleapis.dev/python/pandas-gbq/latest/",
     "issue_tracker": "https://github.com/googleapis/python-bigquery-pandas/issues",
-    "release_level": "beta",
+    "release_level": "preview",
     "language": "python",
     "library_type": "INTEGRATION",
     "repo": "googleapis/python-bigquery-pandas",
     "distribution_name": "pandas-gbq",
     "api_id": "bigquery.googleapis.com",
     "default_version": "",
-    "codeowner_team": "@googleapis/api-bigquery"
+    "codeowner_team": "@googleapis/api-bigquery",
+    "api_shortname": "pandas-gbq"
```

Comment:
The docs you like to say this field is "required if library_type is GAPIC*. Optional for others."

Also, wouldn't `api_shortname` be just `bigquery` so that labeling and such maps to the correct backend API?

Reply:
You're right. Since it's optional, I've removed it to avoid confusion.